### ISSUE TITLE
Feature Request: Override built-in gif picker

### ISSUE THREAD

**Issue by icelander** [moussetc/mattermost-plugin-giphy#34]
Currently the plugin works via slash commands, but the built-in gif chooser still shows (unfiltered) results from Gfycat:

![Screen Shot 2020-09-01 at 3 52 17 PM](https://user-images.githubusercontent.com/131287/91899415-1ec36880-ec6b-11ea-8bff-6c72bfe76d2c.png)

It would be great if the plugin could override this with gifs from the configured gif provider.

**Comment by samzorio** [comment]
any plans to implement this change now that gfycat has permanently shutdown ?

**Comment by moussetc** [comment]
> any plans to implement this change now that gfycat has permanently shutdown ?

No, Mattermost itself is actually planning to switch to Giphy in Mattermost 9.0, according to [the doc](https://docs.mattermost.com/install/deprecated-features.html).

**Comment by moussetc** [comment]
Closing this one, as I do not intend to add a webapp part to the plugin, if even it were technically possible to override the picker (unsure). Currently, the latest versions of Mattermost use Giphy as provider of the picker.
